internal/painter/gl/gl_common.go

```diff
@@ -27,16 +33,6 @@ func logGLError(err uint32) {
 	}
 }
 
-func (p *glPainter) getTexture(object fyne.CanvasObject, creator func(canvasObject fyne.CanvasObject) Texture) Texture {
```

Comment:
I'm not sure I follow why this was moved to platform specific code - it seems the same as before when there was just this 1 instance...

Reply:
Each version depends on a different version of OpenGL to compile. I will look at it more and see if there could be more in common code.

Reply:
I think I managed to get that sorted out somehow.